master/static/srv/proto_get_trial_workloads.sql

```diff
@@ -8,6 +25,9 @@ WITH validations_vt AS (
         jsonb_build_object('avg_metrics', v.metrics->'validation_metrics') as metrics
       FROM validations v
       WHERE v.trial_id = $1
+      AND $4 != 'FILTER_OPTION_CHECKPOINT'
```

Comment:
Added this filter? Think this was left out as a mistake?

Reply:
:+1: